src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line
+import Big from 'big.js';
+
+function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  } if (operation === '-') {
+    return one.minus(two).toString();
+  } if (operation === 'x') {
+    return one.times(two).toString();
+  } if (operation === '÷') {
+    return one.div(two).toString();
```

Comment:
You have not managed division by zero, you can for example display `"INFINITY"` on the display screen
![Screen Shot 2020-03-06 at 4 39 28 PM](https://user-images.githubusercontent.com/40712459/76097993-f236d880-5fc8-11ea-9cb4-ce53a0ee42e9.png)